### PR TITLE
Add Telefónica Icon

### LINKED ISSUE #11455: Request: Telefónica
### Brand Name

Telefónica

### Website

https://www.telefonica.com/

### Popularity Metric

Similiarweb Global Rank: [50,057](https://www.similarweb.com/website/telefonica.com)

### Official Resources for Icon and Color

- SVG: https://commons.wikimedia.org/wiki/File:Telef%C3%B3nica_2021_logo.svg (only “T”)
- Color: `#0066ff`, from above site

### Additional Comments

_No response_

### PR DESCRIPTION
**Pull request by vanshaj-sharma**
image

**Issue:** closes #11455 

**Popularity metric:**
Similiarweb Global Rank: [50,057](https://www.similarweb.com/website/telefonica.com)

### Checklist

- [x] I updated the JSON data in `_data/simple-icons.json`
- [x] I optimized the icon with SVGO or SVGOMG
- [x] The SVG `viewbox` is `0 0 24 24`

I was able to remove the "Telefonica" part in the logo provided at #11455 and also at JSON data, I wrote Telefonica, as Telefónica which was creating issues in linting. Did a simlar sort of a thing in the svg while adding the title tag and also naming the svg. I request the moderators to take a look into this and I will be happy to make changes accordingly if needed

### CODE REVIEW

**Comment by adamrusted** [comment]
@vanshaj-sharma - please tell us what issues you were getting when trying to save with the ó in the name. We can't help if you don't tell us what the issues were :)

File: icons/telefonica.svg [hunk 1]
```diff
@@ -0,0 +1 @@
+<svg role="img" viewBox="0 0 24 24" xmlns="http://www.w3.org/2000/svg"><title>Telefonica</title><path d="M7.162 3.581a3.581 3.581 0 1 0-7.162 0 3.581 3.581 0 0 0 7.162 0m8.419 0a3.581 3.581 0 1 0-7.162 0 3.581 3.581 0 0 0 7.162 0m8.419 0a3.581 3.581 0 1 0-7.162 0 3.581 3.581 0 0 0 7.162 0M15.581 12a3.581 3.581 0 1 0-7.162 0 3.581 3.581 0 0 0 7.162 0m0 8.419a3.581 3.581 0 1 0-7.162 0 3.581 3.581 0 0 0 7.162 0"/></svg>
```
Comment (mondeja) [inline icons/telefonica.svg:1]: If you see the linter message is very clear:

```sh
x icons/telefonica.svg
  x icon-title Unencoded unicode characters found in title "Telefónica": rewrite it as "Telef&#243;nica".
```

File: _data/simple-icons.json [hunk 2]
```diff
@@ -15321,6 +15321,11 @@
             "hex": "1343FB",
             "source": "https://www.telequebec.tv/societe/logo-et-normes-techniques"
         },
+        {
+            "title": "Telefonica",
```
Comment (mondeja) [inline _data/simple-icons.json:15325]: Here you don't need to encode it:

```suggestion
            "title": "Telefónica",
```

**Review by mondeja — CHANGES_REQUESTED** [review]

**Review by adamrusted — APPROVED** [review]
LGTM, thanks @vanshaj-sharma!

**Pull request merged**